props/prop_base.h

```diff
@@ -1376,6 +1376,18 @@ class PropBase : CommandParser, Looper, protected SaberBase, public ModeInterfac
       }
       return true;
     }
+    if (!strcmp(cmd, "swing")) {
+      hybrid_font.PlayPolyphonic(&SFX_swng);
```

Comment:
swing is a lot more complicated than this, as it can be monophonic or polyphonic, swng.wav or swing.wav
Perhaps we should even consider breaking the function in hybrid font into a swing-detection function and a swing playing function, and then we could just call the playing function from here.

Or even better, have the detection function emit an event, and then do the playing in SB_Event2...

Reply:
Hmm. Does this small "perk" feature warrant all that?
Would it suffice to just check if swng OR swing exist and then play one that does with PlayCommon()?

Reply:
I've updated things to a rough start that works. Monophonic stuff needs to be considered next, but how does it look so far now?